fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -565,6 +565,16 @@ protected void addLinkAndOptionsHttpHeaders(final FedoraResource resource) {
         servletResponse.addHeader("Allow", options);
     }
 
+    /**
+     * Add Link and Option headers for a memento
+     * @param path the path of the memento
+     */
+    protected void mementoLinkAndOptionsHttpHeaders(final String path) {
```

Comment:
Since there is nothing in this method that requires or checks that the param `path` is a memento, a clearer method name may be: `addLinkAndOptionsHttpHeadersIfExists`.